examples/ruby-sample/tests/integration_test.rs

```diff
@@ -19,9 +20,9 @@ use std::time::Duration;
 fn basic() {
     IntegrationTest::new("heroku/buildpacks:20", "test-fixtures/simple-ruby-app").run_test(
         |context| {
-            assert!(context.pack_stdout.contains("---> Ruby Buildpack"));
-            assert!(context.pack_stdout.contains("---> Installing bundler"));
-            assert!(context.pack_stdout.contains("---> Installing gems"));
+            assert_contains!(context.pack_stdout, "---> Ruby Buildpack");
+            assert_contains!(context.pack_stdout, "---> Installing bundler");
+            assert_contains!(context.pack_stdout, "---> Installing gems");
 
             context.start_container(&[12345], |container| {
                 std::thread::sleep(Duration::from_secs(1));
```

Comment:
There's another `assert!(....contains(...))` usage that could be replaced with `assert_contains!` below.

Reply:
Thanks!